app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -459,5 +460,63 @@ def clean_up_extra_flavor_keys
     def add_instance_disk(disks, size, location, name)
       super(disks, size, location, name, "openstack")
     end
+
+    def get_services
```

Comment:
@blomquisg yeah the API call for get_services gets only subset of the services, with the OpenStack enabled/disabled info